cardano-coin-selection.cabal

```diff
@@ -48,17 +71,16 @@ library
     , text
     , transformers
   hs-source-dirs:
-      src
+      src/library
   exposed-modules:
       Cardano.CoinSelection
       Cardano.CoinSelection.LargestFirst
       Cardano.CoinSelection.Migration
       Cardano.CoinSelection.RandomImprove
       Cardano.Fee
       Cardano.Types
-      Numeric.Rounding
 
-test-suite unit
+test-suite cardano-coin-selection-test
```

Comment:
when it comes to me I am used to `unit` , `benchmark`, `integration` ... and I like them as they convey what they are in one standarized word

Reply:
>  when it comes to me I am used to unit , benchmark, integration ... and I like them as they convey what they are in one standarized word

It's certainly not fixed in stone.

Though since we already have:
* `cardano-coin-selection`
* `cardano-coin-selection-internal`

It seemed consistent to use:
* `cardano-coin-selection-test`

But we could change this.